configs/KB2040/CMakeLists.txt

```diff
@@ -0,0 +1,3 @@
+#add_executable(KB2040
```

Comment:
This file is not necessary. I know that there is such a file in the `configs/Pico` directory but that one is probably unnecessary as well.